brianlong/www.validators.app
# archive script

#### What's this PR do?
- create script that archives all blockchain models
- 
#### How should this be manually tested?
- run migrations
- run tests
- populate your db with slots/blocks/transactions
- run rails r script/archives/archive_blockchain.rb and make sure it moves records to archive 

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187221917)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
